docs/source/contributing/contributing.md

```diff
@@ -1,45 +1,86 @@
+
+    Very easy feature requests might be updates to the documentation to clarify steps for new users.
+    Harder feature requests may be to add new functionality to the project and will need more in depth discussion about who can complete and maintain the work.
 
-These are not hard rules to be enforced by 🚓 but instead guidelines
-to help you make a contribution.
-
-* prefix the title of your pull request with `[MRG]` if the contribution
-  is complete and should be subjected to a detailed review;
-* create a PR as early as possible, marking it with `[WIP]` while you work on
-  it (good to avoid duplicated work, get broad review of functionality or API,
-  or seek collaborators);
-* a PR solves one problem (do not mix problems together in one PR) with the
-  minimal set of changes;
-* describe why you are proposing the changes you are proposing;
-* try to not rush changes (the definition of rush depends on how big your
-  changes are);
-* Enter your changes into the [changelog](./../../changelog) in `docs/source/changelog.rst`;
-* someone else has to merge your PR;
-* new code needs to come with a test;
-* apply [PEP8](https://www.python.org/dev/peps/pep-0008/) as much
-  as possible, but not too much;
-* no merging if travis is red;
-* do use merge commits instead of merge-by-squashing/-rebasing. This makes it
-  easier to find all changes since the last deployment `git log --merges --pretty=format:"%h %<(10,trunc)%an %<(15)%ar %s" <deployed-revision>..`
+    Feature requests are a great opportunity for you to advocate for the use case you're suggesting.
+    They help others understand how much effort it would be to integrate the work,and - if you're successful at convincing them that this effort is worth it - make it more likely that they to choose to work on it with you.
+
+2. Open an issue.
+  Getting consensus with the community is a great way to save time later.
+3. Make edits in [your fork](https://help.github.com/en/articles/fork-a-repo) of the [repo2docker repository](https://github.com/jupyter/repo2docker).
+4. Make a [pull request](https://help.github.com/en/articles/about-pull-requests).
+Read the [next section](#guidelines-to-getting-a-pull-request-merged) for guidelines for both reviewers and contributors on merging a PR.
+5. Edit [the changelog](./../../changelog) by appending your feature / bug fix to the development version.
+6. Wait for a community member to merge your changes.
+7. (optional) Deploy a new version of repo2docker to mybinder.org by [following these steps](http://mybinder-sre.readthedocs.io/en/latest/deployment/how.html)
+
+## Guidelines to getting a Pull Request merged
 
+These are not hard rules to be enforced by 🚓 but they are suggestions written by the repo2docker maintainers to make getting your contribution completed a smooth process for you and for them.
```

Comment:
```suggestion
These are not hard rules to be enforced by 🚓 but they are suggestions written by the repo2docker maintainers to make getting your contribution completed and smooth process for you and for them.
```
I think this was the missing word.

Reply:
Ah - I think we're parsing the sentence differently.

How about:

```suggestion
These are not hard rules to be enforced by 🚓 but they are suggestions written by the repo2docker maintainers to help complete your contribution as smoothly as possible for both you and for them.
```

Reply:
I like it!

Reply:
I like it.